endoreg_db/views/pdf/pdf_stream.py

```
@@ -0,0 +1,187 @@
+                        logger.warning(
+                            f"Invalid range start {start} for file size {file_size}"
+                        )
+                        raise Http404("Invalid range")
+
+                    if end >= file_size:
+                        end = file_size - 1
+
+                    chunk_size = end - start + 1
+
+                    try:
+                        file_handle = open(file_path, "rb")
```

Pylint (4.0.2)

[refactor] Consider using 'with' for resource-allocating operations

(R1732)